app/components/Gallery.js

```diff
@@ -41,13 +42,26 @@ export default class Gallery extends Component {
     return Math.floor(Math.random()*(max-min+1)+min);
   }
 
+  settingsFromBreakpoint() {
+    if (this.props.breakpoint === 'small') {
+      return {
+        variableWidth: this.randomNumberFromRange(95, 100),
+        nudgeDistance: '10%'
+      } 
+    }
+    return {
+      variableWidth: this.randomNumberFromRange(50, 90),
+      nudgeDistance: '20%'
+    } 
+  }
+
```

Comment:
Thoughts on this? Goal was to not have to keep writing `if (this.props.breakpoint === 'small')` a bunch of times.

Might be a better way of handling this tho..?